backend/typescript/models/user.model.ts

```diff
@@ -3,15 +3,24 @@ import { Role } from "../types";
 
 @Table({ tableName: "users" })
 export default class User extends Model {
+  @Column({ type: DataType.INTEGER })
```

Comment:
I believe the createdAt, updatedAt and id fields can be auto-generated by sequelize if you don't specify them: 
![image](https://user-images.githubusercontent.com/51849738/135741250-b7f786ae-fcc3-4846-b464-2609c2380f1a.png)
So try removing them and verifying that you can find these attributes. May need to test it by creating a row of data using sequelize